src/components/molecules/news-and-events-more/news-and-events-more.scss

```
@@ -0,0 +1,16 @@
+@use "../../variables" as *;
+
+a {
+  font-size: $fontsize-s;
+  color: $black;
+  transition: all 0.2s ease;
+
+  & > i {
+    margin-left: 16px;
+  }
+
+  &:hover {
+    color: $gray_nobel;
+    text-decoration: none;
+  }
+}
```

```suggestion
.news-and-events-more {
  font-size: $fontsize-s;
  color: $black;
  transition: color 0.2s ease;

  & > i {
    margin-left: 16px;
  }

  &:hover {
    color: $gray_nobel;
    text-decoration: none;
  }
}
```
要素セレクタを使うと、news-and-events-more以外のパーツで使われているaタグにも同じスタイルが当たってしまうのでクラスセレクタを使いましょう！

また、これはたいして重要ではないのですが、transtionの対象は指定したほうがコードの意図が読み取りやすくて望ましいです。

---

ありがとうございます！！修正しました。
iタグのクラス名はスペースがあるので要素セレクタのままでも可でしょうか？

---

よい質問だと思います！
i の要素セレクタに関しては、今回入れ子にしてあるのでそのままにしました。
というのも、今回のコードは scss から css に変換すると

```
.news-and-events-more {
  font-size: $fontsize-s;
  color: $black;
  transition: color 0.2s ease;
}
.news-and-events-more > i {
  margin-left: 16px;
}
.news-and-events-more:hover {
  color: $gray_nobel;
  text-decoration: none;
}
```
こんな感じになります。（本当は変数も変換されるのですが、確認の手間を惜しんでいます。詳しくは新課題ページの[ここ](https://shinonome.io/frontend-basic/lastAssignment/storybook_sass/)をご覧ください）  
`i` は要素セレクタですが、子結合子によって `.news-and-events-more` の子要素の `i` と限定されているので、他の要素に影響を及ぼすことが考えにくいため大丈夫ということです。